docs/user-docs/latest/plugins/strapi-plugins.md

```diff
@@ -0,0 +1,32 @@
+
+### <img width="28" src="../assets/plugins/icon_i18n-plugin.png"> Internationalization plugin
+
+The Internationalization plugin is an additional plugin that is not installed by default. It can be installed through the Marketplace (see [Installing plugins via the Marketplace](installing-plugins-via-marketplace.md)).
+
+This plugin allows to manage content in different languages, called "locales". With the Internationalization plugin, it is possible to:
+
+- define what locales should be available in the Strapi application,
+- define which content-types and fields can be translated in different locales, or should only be available in the default locale,
+- translate content and manage it each locale at a time.
+
+The Internationalization plugin impacts several parts of the admin panel. The table below lists all the additional options and settings that are added to a Strapi application once the plugin has been installed.
+
+| Section impacted | Options and settings                                                                                    |
+|------------------|---------------------------------------------------------------------------------------------------------|
+| Settings         | <ul><li>Addition of a new "Internationalization" setting sub-section, from which to add, edit or delete locales available for the application (see [Configuring Internationalization locales](../settings/managing-global-settings.md#configuring-internationalization-locales)). <br> 👉 Path reminder: *General > Settings > Global Settings > Internationalization* </li> <br> <li>Addition of new permissions for administator roles: access to content-types, as well as possible actions on the content-types, can be defined depending on the locale (see [Configuring role's permissions](/user-docs/latest/users-roles-permissions/configuring-administrator-roles.md#configuring-role-s-permissions)). <br> 👉 Path reminder: *General > Settings > Administration panel*</li></ul> |
+| Content-Types Builder | <ul><li>Addition of a new setting at content-type level, to allow or not localisation/translation of the content-type (see [Creating a new content-type](/user-docs/latest/content-types-builder/creating-new-content-type.md#creating-a-new-content-type)).</li> <li>Addition of a new setting at field level, to allow or not localisation/translation of the content-type (see [Configuring fields for content types](/user-docs/latest/content-types-builder/configuring-fields-content-type.md#regular-fields)).</li></ul> |
+| Content Manager | <ul><li>Addition of new *Locales* filter in collection types list view, to manage entries per locale (see [Introduction to the Content Manager](/user-docs/latest/content-manager/introduction-to-content-manager.md#collection-types)).</li> <li>Addition of new options in content-types edit view, to translate content and manage it per locale (see [Translating content](/user-docs/latest/content-manager/translating-content.md)).</li></ul> |
```

Comment:
Here I have a small concern with the word "filter", do you think it could be understood as a new filtering parameter added to the existing filter system, instead of a new component to choose the locale to display?

Not a big deal

Reply:
oh actually one more question, we apply limitations to relations displayed in the edit view when i18n is activated on a CT (eg. sometimes users will only see relations in the default locale or in the current locale the user is browsing) should we mention it somewhere?

Reply:
For the filter concern, I have re-read it and I think it will be alright. In the Content Manager documentation, there is a clear difference between the regular filters (which are fully documented) and the new i18n option to filter by locale, with a screenshot to reinforce that.
For the relations limitations, I've added a note in Content Manager > Managing relational fields.